js/trackers.js

```diff
@@ -54,6 +55,11 @@ function isTracker(urlToCheck, currLocation, tabId, request) {
             blockSettings.push('Social');
         }
 
+        var whitelistedTracker = checkWhitelist(urlToCheck, currLocation, request)
+        if (whitelistedTracker) {
+            return whitelistedTracker
+        }
```

Comment:
You have a property called `isWhiteListed` on line 46. You don't seem to be doing anything with it - ? Seems like that should be removed.

Reply:
Please also see lines 27-29 in this file. This seems like an opportune time to move that `duckduckgo.com` check to the open source repo instead of letting it be a special one-off check here.

Reply:
There is also a `var toBlock = false` defined in the `isTrackers()` function. But within `isTrackers` you never do anything with it except return it (unchanged) as false at the end. This would be an opportune time to clean all of this type of stuff in this file up - in some functions within this file you define a `toBlock` which you do use, and in others you are calling the same thing something different. Please unify approach and clarify all of this, refactor and make legible. The code in this file is pretty hard to follow otherwise.